pip_audit/_service/pypi.py

```diff
@@ -109,12 +109,11 @@ def _get_pip_cache() -> str:
 
 
 def _get_cache_dir(custom_cache_dir: Optional[Path]) -> str:
-    pip_cache_dir: Optional[str] = (
-        _get_pip_cache() if _PIP_VERSION >= _MINIMUM_PIP_VERSION else None
-    )
     if custom_cache_dir is not None:
         return str(custom_cache_dir)
-    elif pip_cache_dir is not None:  # pragma: no cover
+    elif (
+        pip_cache_dir := _get_pip_cache() if _PIP_VERSION >= _MINIMUM_PIP_VERSION else None
```

Comment:
We're trying to support Python 3.6 and later so we'll need to avoid the `:=`.

Reply:
Fixed in: https://github.com/trailofbits/pip-audit/pull/161/commits/00ead12c134fb0315b4ef6e3026c81823ab0ab7b